publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/identifiers/HandleIdentifierEventHandler.java

```diff
@@ -125,8 +124,7 @@ protected Void processInputPayload(EventReference input) {
             var resourceUpdate = parseResourceUpdateInput(eventBlob);
             if (isPublished(resourceUpdate) && isMissingHandle(resourceUpdate)) {
                 logger.info("Creating handle for publication: {}", resourceUpdate.getIdentifier());
-                var userInstance = UserInstance.create(resourceUpdate.getOwner(), resourceUpdate.getCustomerId());
-                var publication = fetchPublication(userInstance, resourceUpdate.getIdentifier());
+                var publication = fetchPublication(resourceUpdate.getIdentifier());
```

Comment:
No need to fetch publication using user instance when you have publication identifier. Using "simple" fetch method instead.